[2024-01-02T17:45:04.097+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 24_01_02_fx_mkt_data.first_attempt manual__2024-01-02T17:44:58.908831+00:00 [queued]>
[2024-01-02T17:45:04.132+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 24_01_02_fx_mkt_data.first_attempt manual__2024-01-02T17:44:58.908831+00:00 [queued]>
[2024-01-02T17:45:04.133+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2024-01-02T17:45:04.188+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): first_attempt> on 2024-01-02 17:44:58.908831+00:00
[2024-01-02T17:45:04.206+0000] {standard_task_runner.py:57} INFO - Started process 1814 to run task
[2024-01-02T17:45:04.216+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', '24_01_02_fx_mkt_data', 'first_attempt', 'manual__2024-01-02T17:44:58.908831+00:00', '--job-id', '202', '--raw', '--subdir', 'DAGS_FOLDER/get_fx_mkt_data.py', '--cfg-path', '/tmp/tmpq59m91mv']
[2024-01-02T17:45:04.223+0000] {standard_task_runner.py:85} INFO - Job 202: Subtask first_attempt
[2024-01-02T17:45:04.491+0000] {task_command.py:416} INFO - Running <TaskInstance: 24_01_02_fx_mkt_data.first_attempt manual__2024-01-02T17:44:58.908831+00:00 [running]> on host ff92cbb9a252
[2024-01-02T17:45:04.998+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='24_01_02_fx_mkt_data' AIRFLOW_CTX_TASK_ID='first_attempt' AIRFLOW_CTX_EXECUTION_DATE='2024-01-02T17:44:58.908831+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-02T17:44:58.908831+00:00'
[2024-01-02T17:45:05.003+0000] {oandapyV20.py:207} INFO - setting up API-client for environment practice
[2024-01-02T17:45:06.286+0000] {logging_mixin.py:154} INFO - Error processing AUD_USD: strptime() argument 1 must be str, not datetime.datetime
[2024-01-02T17:45:06.288+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-02T17:45:06.350+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=24_01_02_fx_mkt_data, task_id=first_attempt, execution_date=20240102T174458, start_date=20240102T174504, end_date=20240102T174506
[2024-01-02T17:45:06.436+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T17:45:06.522+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
